contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -206,15 +206,17 @@ def _check_and_update_duration(self, storage_key: str, current_duration: float,
                     "Last run duration: {last:.2f}s, current run duration: {cur:.2f}s".format(
                         expected=expected_duration, last=last_duration, cur=current_duration))
 
+
 class WaitForLoadingSymbolsAndCheckModule(E2ETestCase):
     """
     Waits for automatically loading all symbol files and checks if the specified module was loaded
     successfully.
     """
     def _execute(self, module_search_string: str):
-        WaitForLoadingSymbolsAndVerifyCache()
+        WaitForLoadingSymbolsAndVerifyCache().execute(self.suite)
```

Comment:
I would split the commits between this line (the proper fix) and the nice refactor (http://b/236965643). Since you've already created this PR, consider at least to explain that this line is the needed fix and the rest is part of the other bug, what do you think?

Reply:
Well I *think* that this line fixes the issue already, but I started the changes in the loading states first and then discovered this, so I'm not 100% sure.

Reply:
Also I wanted to copy the PR description into the commit message which explains that this is the main fix, wdyt?